tutorials/ngsi_v2/fiware_env/docker-compose.yml

```diff
@@ -0,0 +1,96 @@
+
+volumes:
+    mongodb: ~
+    crate-db: ~
+
+services:
+    iot-agent:
+        image: fiware/iotagent-json:1.26.0
+        hostname: iot-agent
+        container_name: fiware-iot-agent
+        depends_on:
+            - mongodb
+            - mosquitto
+        expose:
+            - "4041"
+        ports:
+            - "4041:4041"
+        environment:
+            - "IOTA_CB_HOST=orion"
+            - "IOTA_CB_PORT=1026"
+            - "IOTA_NORTH_PORT=4041"
+            - "IOTA_REGISTRY_TYPE=mongodb"
+            - "IOTA_MONGO_HOST=mongodb"
+            - "IOTA_MONGO_PORT=27017"
+            - "IOTA_MONGO_DB=iotagent-json"
+            - "IOTA_PROVIDER_URL=http://iot-agent:4041"
+            - "IOTA_MQTT_HOST=mosquitto"
+            - "IOTA_MQTT_PORT=1883"
+            - "IOTA_DEFAULT_RESOURCE="
```

Comment:
The best way is not to rely on the default value. It would make more sense to set the resource in the Service Group itself. But I think "iot/json" is a better default value.